### PR TITLE
289 refacto simplification remove registers

### PR DESCRIPTION
**Pull request by glannuzel**
Register class is no longer alive!

Attributes are accessed by properties

### CODE REVIEW

File: src/reachy2_sdk/orbita/orbita.py [hunk 1]
```diff
@@ -107,40 +104,37 @@ def turn_on(self) -> None:
...
+        fut = asyncio.run_coroutine_threadsafe(set_in_loop(), self._loop)
+        fut.result()
 
     def is_on(self) -> Any:
         """Get compliancy of the actuator"""
         return not self._compliant
 
     @property
-    def temperatures(self) -> Dict[str, Register]:
+    def temperatures(self) -> Dict[str, Any]:
```
Comment (FabienDanieau) [inline src/reachy2_sdk/orbita/orbita.py:134]: Avoid the use of Any. mypy will do a better job knowing the exact type. I believe we have a float here

File: src/reachy2_sdk/orbita/orbita.py [hunk 2]
```diff
@@ -107,40 +104,37 @@ def turn_on(self) -> None:
...
+        self._state["compliant"] = True
+
+        async def set_in_loop() -> None:
+            self._register_needing_sync.append("compliant")
+            self._need_sync.set()
+
+        fut = asyncio.run_coroutine_threadsafe(set_in_loop(), self._loop)
+        fut.result()
 
     def is_on(self) -> Any:
```
Comment (FabienDanieau) [inline src/reachy2_sdk/orbita/orbita.py:129]: Return a bool, not Any

File: src/reachy2_sdk/orbita/utils.py [hunk 3]
```diff
@@ -20,3 +22,32 @@ def to_internal_position(pos: float) -> Any:
         return np.deg2rad(pos)
     except TypeError:
         raise TypeError(f"Excepted one of: int, float, got {type(pos).__name__}")
+
+
+def unwrapped_proto_value(value: Any) -> Any:
```
Comment (FabienDanieau) [inline src/reachy2_sdk/orbita/utils.py:27]: Same here, avoid the Any is possible

**Review by FabienDanieau — COMMENTED** [review]
Tests are ok with the fake robot.
I think we can improve few stuff during the factorization step. If you already addressed my comments in the sub branches you can go ahead and merge. Otherwise I can also help to make these changes

File: src/reachy2_sdk/orbita/orbita3d.py [hunk 4]
```diff
@@ -193,23 +194,38 @@ def _pop_command(self) -> Orbita3dCommand:
 
         return command
 
-    def _update_with(self, new_state: Orbita3dState) -> None:
+    def _update_with(self, new_state: Orbita3dState) -> None:  # noqa: C901
```
[the remark this answers is not in the document]
Comment (glannuzel) [reply src/reachy2_sdk/orbita/orbita3d.py:197]: Can I do this modification in #298, which deals with classes refacto ?

File: src/reachy2_sdk/orbita/utils.py (hunk 3 above)
Comment (glannuzel) [reply src/reachy2_sdk/orbita/utils.py]: Function removed

**Review by FabienDanieau — APPROVED** [review]

**Pull request merged**